src/project/FileTreeView.js

```diff
@@ -31,8 +31,8 @@
 define(function (require, exports, module) {
     "use strict";
 
-    var React             = require("thirdparty/react"),
-        ReactDOM          = require("thirdparty/react-dom"),
+    var React             = require("thirdparty/preact"),
```

Comment:
We probably should just `find-and-replace` all the React and ReactDOM mentions to `Preact`